tests/unit_tests/test_unit.py

```diff
@@ -418,18 +416,6 @@ def test_is_valid_url():
     assert not casanovo._is_valid_url("foobar")
 
 
-def test_tensorboard():
```

Comment:
Since this branch uses PyLighting's implementation of tensorboard, I have removed this test.